app/src/main/java/com/example/unsplashphotos/domain/usecase/PhotoUseCase.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.unsplashphotos.domain.usecase
+
+
+import androidx.paging.Pager
+import androidx.paging.PagingConfig
+import androidx.paging.PagingData
+import androidx.paging.PagingSource
+import com.example.unsplashphotos.data.model.Photo
+import kotlinx.coroutines.flow.Flow
+import javax.inject.Inject
+
+
+class PhotoUseCase @Inject constructor(private val photoPagingSource: PagingSource<Int, Photo>) {
```

Comment:
Typically, UseCase name should start with a verb that represents some kind of business flow. For example, FetchPhotosUseCase, UpdateUserInfoUseCase etc.
Please read these two articles on UseCases. (Good to see that you already have the basic understanding of UseCase. 👍) 
1. https://www.techyourchance.com/how-to-use-case-interactor-java/
2. https://www.techyourchance.com/how-to-use-case-interactor-kotlin/